### PR TITLE
Fix hlint

### PR DESCRIPTION
**Pull request by MHova**

### CODE REVIEW

File: canteven-http.cabal [hunk 1]
```diff
@@ -40,6 +40,8 @@ library
     uuid             >= 1.3.12   && < 1.4,
     wai              >= 3.2      && < 3.3,
     wai-extra        >= 3.0.8    && < 3.1
-  hs-source-dirs:      src
+  hs-source-dirs:
+    src
+    .src
```
Comment (potomak) [inline canteven-http.cabal]: Why a hidden directory?

Comment (MHova) [inline canteven-http.cabal]: Because HLint automatically skips over `.` directories.

**Comment by taphu** [comment]
This is a *very* strange fix in my opinion. I think I would have been more inclined to fix our build process in order to have more fine-grained control over hlint invocation.

:+1: for creativity.

**Comment by MHova** [comment]
Ha ha. Well, this is also a generalized fix for any poor soul who wants to clone this open-source project.

**Pull request merged**